File: src/main/java/io/bloco/faker/components/Team.java
package io.bloco.faker.components;

import io.bloco.faker.FakerComponent;
import io.bloco.faker.FakerData;

public class Team extends FakerComponent {

    public Team(FakerData data) {
        super(data);
    }

    public String name() {
        return parse(fetch("team.name"));
    }

    public String creature() {
        return fetch("team.creature");
    }

    public String state() {
        return call("address.state");
    }

    public String sport() {
        return fetch("team.sport");
    }
}

File: lib/src/main/kotlin/io/bloco/faker/components/Team.kt
package io.bloco.faker.components

import io.bloco.faker.FakerComponent
import io.bloco.faker.FakerData

class Team(data: FakerData) : FakerComponent(data) {
    fun name(): String {
        return parse(fetch("team.name"))
    }

    fun creature(): String {
        return fetch("team.creature")
    }

    fun state(): String {
        return call("address.state")
    }

    fun sport(): String {
        return fetch("team.sport")
    }
}